src/Core/Scheduler/Cache.js

```diff
@@ -61,16 +61,19 @@ class Cache {
     get(key1, key2, key3) {
         const entry_1 = this.data.get(key1);
         if (entry_1 == undefined) {
-            // eslint-disable-next-line
             return;
+        } else if (!(entry_1 instanceof Map)) {
```

Comment:
Not a big fan of `instanceof`, won't it slow too much things ? But I don't really see an alternative, besides a ugly `entry_1.constructor.name == 'Map'`.

Reply:
I replace by `entry_1.lastTimeUsed != undefined`